Library/Homebrew/commands.rb

```diff
@@ -203,7 +203,7 @@ def command_description(command, short: false)
 
     if (cmd_parser = Homebrew::CLI::Parser.from_cmd_path(path))
       if short
-        cmd_parser.description.split(".").first
+        cmd_parser.description.split(/\.(?>\s|$)/).first
```

Comment:
Should this be
```suggestion
        cmd_parser.description.split(/\.(?=\s|$)/).first
```
? Though not sure it really makes a difference.

Reply:
Yea, shouldn't make a difference here.

I used `?>` to make `(\s|$)` a non-capturing group. Without that, [the group will be included in the result array too](https://ruby-doc.org/core-2.6.10/String.html#method-i-split).

```
brew(main):001:0> desc = "URL https://docs.brew.sh. Hello world. Path ~/.nodenv."
=> "URL https://docs.brew.sh. Hello world. Path ~/.nodenv."
brew(main):002:0> desc.split(/\.(\s|$)/)
=> ["URL https://docs.brew.sh", " ", "Hello world", " ", "Path ~/.nodenv"]
brew(main):003:0> desc.split(/\.(?>\s|$)/)
=> ["URL https://docs.brew.sh", "Hello world", "Path ~/.nodenv"]
```

But `?=` (positive lookahead) does not actually consume the whitespace. Consequently, the space can still be found in subsequent sentences.

```
brew(main):004:0> desc.split(/\.(?=\s|$)/)
=> ["URL https://docs.brew.sh", " Hello world", " Path ~/.nodenv"]
```

So, in terms of sentence splitting, `?>` looks more appropriate.